docs/settings/banners-settings.asciidoc

```diff
@@ -16,7 +16,7 @@ You can configure the `xpack.banners` settings in your `kibana.yml` file.
 |===
 
 | `xpack.banners.placement`
-| Set to `header` to enable the header banner. Defaults to `disabled`.
```

Comment:
You'll also need to add documentation about the Banner settings to the [Advanced Settings](https://www.elastic.co/guide/en/kibana/current/advanced-options.html) page.

Reply:
Thanks. WDYT about https://github.com/elastic/kibana/pull/94449/commits/14c33b6398ea918dfb62fdac9e7c185a3349db52?